init-tools.sh

```diff
@@ -96,7 +96,11 @@ if [ ! -e "$__DOTNET_PATH" ]; then
                     exit 1
                 fi
                 ;;
-
+            FreeBSD)
+                ulimit -n 2048
```

Comment:
The ulimit should not be needed for FreeBSD. There was a bug for OSX fixed a year ago that required the ulimit to be set, but it didn't affect other platforms. We can get rid of it for OSX too if the managed build tools already contain the fix, which might be the case now.